documentation/modules/rn-2.2.adoc

```diff
@@ -98,3 +98,7 @@ If a Plan CR references storage, network, or VMs by name instead of by ID, the r
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.If a target VM resource is deleted during the image conversion stage, the migration of the VM is still marked as successful.
```

Comment:
1. If **the** target VM resource is deleted - to make clear it is the migrated VM in the target side
2. I think "VM migration is still marked as successful." is better